backon/src/lib.rs

```diff
@@ -33,14 +33,42 @@
 //!
 //! Retry in BackON requires an implementation for sleeping. BackON will accept a [`Sleeper`] to pause for a specified duration.
 //!
-//! BackON employs the following default sleep implementations:
+//! ## Default `Sleeper`
 //!
-//! - `tokio-sleep`: Utilizes [`TokioSleeper`] within a Tokio context in non-wasm32 environments.
-//! - `gloo-timers-sleep`: Utilizes [`GlooTimersSleep`] to pause in wasm32 environments.
+//! Currently, BackON has 2 built-in `Sleeper` implementations for different environments,
```

Comment:
Should be `3`, we have `std-blocking-sleep` for blocking context.

Reply:
Added the `BlockingSleeper` to the table.